source/gui/configProfiles.py

```diff
@@ -184,15 +184,15 @@ def onNew(self, evt):
 
 	def onDelete(self, evt):
 		index = self.profileList.Selection
+		name = self.profileNames[index]
 		if gui.messageBox(
 			# Translators: The confirmation prompt displayed when the user requests to delete a configuration profile.

```

Comment:
It is good to inform translators of any substitution parameters. 

```suggestion
			# Translators: The confirmation prompt displayed when the user requests to delete a configuration profile.
			# The placeholder {} is replaced with the name of the configuration profile that will be deleted.
```